app/controllers/api/map_controller.rb

```diff
@@ -92,13 +91,16 @@ def index
 
       # this "uniq" may be slightly inefficient; it may be better to first collect and output
       # all node-related relations, then find the *not yet covered* way-related ones etc.
+      @relations = []
```

Comment:
`@relations = relations.uniq`
 
Or perhaps rename the `relations` variable right from the start to `@relations`, and then `@relations.uniq!` at this point